front/src/modules/rollingStock/components/RollingStockSelector/SearchRollingStock.tsx

```diff
@@ -160,8 +164,21 @@ const SearchRollingStock = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [filters, rollingStocks]);
 
+  useEffect(() => {
+    if (isDupplicating) {
+      setFilters({
```

Comment:
Can you set this in a new function resetFilters ?

Reply:
done